plugin-common/src/main/kotlin/com/gradleup/gr8/Gr8Configurator.kt

```diff
@@ -158,7 +158,7 @@ open class Gr8Configurator(
 
       val classPathConfiguration = project.configurations.getByName(classPathConfiguration.get())
       classPathFiles.from(classPathConfiguration.filter {
-        !stripGradleApi || !it.isGradleApi()
+        (!stripGradleApi || !it.isGradleApi()) && !it.name.endsWith(".pom")
```

Comment:
Feels weird to have pom files in a configuration. Do you have a reproducer? If I dump the `compileClasspath` of `test-plugin`, I get this:

```
 
afterEvaluate {
  println(configurations.named("compileClasspath").files)
}

org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.5.21/6b3de2a43405a65502728047db37a98a0c7e72f0/kotlin-stdlib-jdk8-1.5.21.jar
dev.gradleplugins/gradle-api/7.6/e00fd237e23eb7c5c23132f17cdc394defa12def/gradle-api-7.6.jar
org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.5.21/f059658740a4b3a3461aba9681457615332bae1c/kotlin-stdlib-jdk7-1.5.21.jar
org.jetbrains.kotlin/kotlin-stdlib/1.5.21/2f537cad7e9eeb9da73738c8812e1e4cf9b62e4e/kotlin-stdlib-1.5.21.jar
org.codehaus.groovy/groovy/3.0.13/7652b22000c2cebbfdcfdf530a64e41baa518cb/groovy-3.0.13.jar
org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar
org.jetbrains.kotlin/kotlin-stdlib-common/1.5.21/cc8bf3586fd2ebcf234058b9440bb406e62dfacb/kotlin-stdlib-common-1.5.21.jar
```

Reply:
Arg, never mind, I dumped the wrong configuration, I can see it now 👍